### DISCUSSION TITLE
Q about your extension

### DISCUSSION THREAD

**Discussion by mucaadciiltire** [sharafdin/markdown-master#4] [Q&A] [answered]
when will the extension release?

**Comment by sharafdin** [comment, marked as answer]
Dear Mr. Mucaad,

I would like to express my sincere gratitude for your support in the upcoming extension project. While we do not have an exact launch time at this moment, rest assured that it will be initiated in the near future, Insha'Allah. Your assistance has been invaluable, and we are excited about the imminent progress.

Thank you once again for your dedication and contribution to this project.

Sincerely,
Mr Sharafdin
